### ISSUE TITLE
Error adding repository

### ISSUE THREAD

**Issue by sapeg** [pantheon-tweaks/pantheon-tweaks#190]
When executing `sudo add-apt-repository -y ppa:philip.scott/pantheon-tweaks`, an error appears:
```
E: Repository "https://ppa.launchpadcontent.net/philip.scott/elementary-tweaks/ubuntu jammy Release" does not contain a Release file.
N: Updating from this repository cannot be done in a secure way, so it is disabled by default.
```
Elementary OS 7.1 Horus
Ubuntu 22.04.3 LTS
Linux 6.5.0-15-generic

**Comment by ryonakano** [comment]
elementary Tweaks only supports elementary OS Hera or below. You should use Pantheon Tweaks instead on 7.1 Horus, as per written in our [README](https://github.com/pantheon-tweaks/pantheon-tweaks/blob/dev/README.md).

**Comment by sapeg** [comment]
The issue is closed. The solution is [here](https://www.linuxslaves.com/2021/01/install-pantheon-tweaks-elementary-os-6-odin.html)

**Comment by sapeg** [comment]
The issue is closed 07.02.2024, 17:20, "Ryo Nakano" ***@***.***>: elementary Tweaks only supports elementary OS Hera or below. You should use Pantheon Tweaks instead on 7.1 Horus, as per written in our README.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением, Павел Макарчук.
